client/src/components/GreensockComponent/GreensockComponent.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const GreensockComponent = ({ message }) => {
+  return <section className="Greensock_component">{message}</section>;
+};
+
+GreensockComponent.propTypes = {
+  message: PropTypes.string
```

Comment:
Let's add a default prop if we are not marking prop types as required.

Reply:
currently, the placeholder text we are using to depict the mockup, is defined as a default property called `placeHolder`. 
the property `message`, is from the react app bootstrapped/generated code, which is a part of the redux store. I have not deleted the generated properties. can do so if need be.

What I did was, copy the sample container over, and create three containers similar to it. Hence, the new containers have the react generated defaults too. I was not sure, if we need them or not, but the generated code, serves as an example for using redux stores. 
so left the message and sample properties untouched.

Reply:
Okay. We can ignore this for now. This was added by me though the sample container and stuff. But it's better to add something like this:
```js
GreensockComponent.defaultProps = {
  message: '',
};
```
when we don't use `isRequired` for any proptype in props validation.